src/gui/torrentcontentmodel.cpp

```diff
@@ -52,13 +50,8 @@
 #include "torrentcontentmodelitem.h"
 #include "torrentcontentmodelfolder.h"
 #include "torrentcontentmodelfile.h"
-
-#ifdef Q_OS_MAC
-struct NSImage;
-// This function is a private QtGui library export on macOS
-// See src/gui/painting/qcoregraphics_p.h for more details
-// QtMac::fromCGImageRef takes a CGImageRef and thus requires a double conversion
-QPixmap qt_mac_toQPixmap(const NSImage *image, const QSizeF &size);
+#if defined(Q_OS_MAC) 
```

Comment:
I would prefer `#ifdef Q_OS_MAC` as it was like that before your edit.
The rest of the file is a bit inconsistent.